design-decisions/2018-09-20-Decision-Template.md

```diff
@@ -0,0 +1,64 @@
+Decision Template
+=================
+
+Background
+----------
+
+We are writing decision documents for each decision that we want to record.
+These live in the `design-decisions` directory, using file names with
+the `yyyy-mm-dd-<dash-separated-title>.md` format, with the date
+being roughly the date when the decision was made.
+
+Currently (2018-09-19) the document format is unrestricted, but loosely
```

Comment:
Now that this file has a date in the name, we probably don't need to specify the date in the text.